src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangle extends Figure implements AreaCalculator, FigureDraw{

```

Comment:
```suggestion
public class RightTriangle extends Figure {

```
ці інтерфейси може імплементувати клас Figure, тоді не потрібно буде прописувати їх в кожному класі насліднику